configs/settings.yml

```diff
@@ -26,13 +26,12 @@ base: &base
   cert_manager_cluster_issuer: ${CERT_MANAGER_CLUSTER_ISSUER:-letsencrypt}
   ingress_default_port: 443
   service_checks:
-    ip_timeout: 180s
-    ip_ping_timeout: 600s
-    availability_timeout: 600s
-    await_status_timeout: 600s
+    ip_timeout: 540s
```

Comment:
This variable is deprecated and doesn't use more

Reply:
thanks! dropped. any others?

Reply:
Other settings are still used